### PR TITLE
TPD-1290, TPD-1295, TPD-1306, Retaining pending people list by keeping track of HiddenPeople.

### PR DESCRIPTION
**Pull request by josearmandohf**
[touchpointsoftware.atlassian.net/browse/TPD-1290](https://touchpointsoftware.atlassian.net/browse/TPD-1290)

[https://touchpointsoftware.atlassian.net/browse/TPD-1295](https://touchpointsoftware.atlassian.net/browse/TPD-1295)

[https://touchpointsoftware.atlassian.net/browse/TPD-1306](https://touchpointsoftware.atlassian.net/browse/TPD-1306)

### CODE REVIEW

File: CmsData/API/MemberRules.cs [hunk 1]
```diff
@@ -25,6 +25,8 @@ public void ChangeRecordType(bool toBusiness)
                 LastName = string.Concat(FirstName, " ", LastName);
                 FirstName = "";
                 IsBusiness = true;
+                MemberStatusId = 20;
```
Comment (brondavies) [inline CmsData/API/MemberRules.cs:28]: Use `MemberStatusCode.NotMember`

File: CmsData/API/MemberRules.cs [hunk 2]
```diff
@@ -25,6 +25,8 @@ public void ChangeRecordType(bool toBusiness)
...
+                CellPhone = CellPhone ?? (CellPhone.HasValue() ? CellPhone : WorkPhone);
```
Comment (brondavies) [inline CmsData/API/MemberRules.cs:29]: Simplify both of these to just `CellPhone.HasValue() ? CellPhone : WorkPhone`

File: CmsData/API/MemberRules.cs (hunk 1 above)
Comment (josearmandohf) [reply CmsData/API/MemberRules.cs:28]: Done

File: CmsData/API/MemberRules.cs (hunk 2 above)
Comment (josearmandohf) [reply CmsData/API/MemberRules.cs:29]: Done

**Pull request merged**